keras/src/utils/jax_layer.py

```diff
@@ -650,13 +807,13 @@ def _variables_to_params_and_state(self, variables):
 
     def _get_init_rng(self):
         return {
-            "params": self.seed_generator.next(),
-            "dropout": self.seed_generator.next(),
+            "params": super()._get_init_rng(),
+            "dropout": super()._get_init_rng(),
```

Comment:
Do `self._get_init_rng()` instead. One of the reasons for isolating them like that is to be able to override them.